_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+
+###### Time Management
+
+Teams should not be fully allocated to their enabling function.
+
+Firstly, each team must resolve time for the continuous self-improvement, which should be equal to, or strictly greater than, 20% of their teams available work time.
+
+Secondly, teams are free to self organize under the condition that they are able to meet the needs of the directorate’s priorities. For example, a team within the directorate may accept to fulfill an enabling function for another team within the organization.
+
+Thirdly, an enabling team need not commit all of their time to being an enabling team. For example, in the case of the IT Strategy team they are also responsible for setting the IT vision for the department, and thus cannot dedicate 100%(-20%) of their time to their enabling function. In this particular case, for example, we recommend that the IT Strategy spend a maximum of (100-20)/2 percent of their time on enabling other teams.
+
+##### External Interactions
+
+This is an initial proposal and is intended to allow for iterations to refine and modify the process as needed.
+
+###### Interactions
+
+The book Team Topologies, referenced below, recommends that Enabling Teams work with other teams for a period (potentially a few months) [TO-DO: Quote needed] in order to assist them with onboarding new technologies and approaches, and improving their existing processes and procedures. The goal is to ensure the interactions are brief enough that the teams being enabled do not become dependent on the enabling teams. The process improvements should stand on their own, and should not involve [toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/). That is to say, the new processes and procedures should aim to automate or improve the existing processes, not lend temporary person-power to the teams. Given the short and iterative nature of the interactions between the teams (duration of interactions should be defined by number of sprints) the enabling teams must identify quick wins -- the amount of automation that could be done within the limited time frame that would result in the greatest amount of automated work.
+
+###### Tooling
+
+[TO-DO: Quote from Team Topologies regarding varying levels of team communication]
+
+The amount of collaboration between the enabling teams and the enabled teams should vary significantly during the time which the enabling team is paired with the enabled team. As such, collaboration tools that allow for the rapid creation and disintegration should be leveraged. An example of a viable tool for such a requirement would be Slack, where groups can be created and disposed of with relative ease.
+
+#### Process
+
+##### Teams Iterative Process
```

Comment:
I would add another item:
- If an enabling team has a particular task that has remained on the KANBAN board for too long (e.g. 2 months), that task goes back in the backlog and may be picked up by another enabling team

Reply:
That should be discussed automatically at the sprint planning, every two weeks. If it hasn't been completed, it should probably even be discussed at the retrospective.

Reply:
@remyb2canada 

"picked up by another enabling team" 

hmm .. that would be tricky, I imagine. Possible, but not sure I'd vote in favour of such a process (not commonly, any who). 

Perhaps we could discuss how that mechanic could work... have trouble imaging it atm